core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -236,16 +242,15 @@ object SparkEnv extends Logging {
     val securityManager = new SecurityManager(conf)
 
     // Create the ActorSystem for Akka and get the port it binds to.
-    val (actorSystem, boundPort) = {
-      val actorSystemName = if (isDriver) driverActorSystemName else executorActorSystemName
-      AkkaUtils.createActorSystem(actorSystemName, hostname, port, conf, securityManager)
-    }
+    val actorSystemName = if (isDriver) driverActorSystemName else executorActorSystemName
```

Comment:
Seems like this is still needed because the `MapOutputTracker` stuff hasn't been ported. Is that intentional or were you planning to also port that API to use the new abstraction?